### ISSUE TITLE
Docs - 404

### ISSUE THREAD

**Issue by BartonR0** [esnet/gdg#80]
![image](https://user-images.githubusercontent.com/54040177/160384451-ecfd0005-1963-41a6-a9be-86be2067bef0.png)
via your link here
![image](https://user-images.githubusercontent.com/54040177/160384642-6d164248-5c46-4243-bd69-53efc6ef0a5d.png)

**Comment by safaci2000** [comment]
Yup, sorry about that, moved ownership of repo and that broke a few things.  Try again.  the new link should be https://software.es.net/gdg/

**Comment by safaci2000** [comment]
Didn't do a PR for this one, but for reference, here's the change: https://github.com/esnet/gdg/commit/e5b265f464c5e1e84268f9026f86f7bb49447bf0
